src/sentry/static/sentry/app/views/alerts/list/onboarding.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import ButtonBar from 'app/components/buttonBar';
+import OnboardingPanel from 'app/components/onboardingPanel';
+import emptyStateImg from 'app/../images/spot/alerts-empty-state.svg';
+
+type Props = {
+  actions: React.ReactNode;
+};
+
+function Onboarding({actions}: Props) {
```

Comment:
Please, why did you decide not to use func component?

Reply:
Like `React.FC` type?

Reply:
yes